docker/publish-nanoserver.ps1

```diff
@@ -42,7 +42,7 @@ function PushImages($imageTags) {
 }
 function GetImageTags($repo, $version) {
         return @(
-            #"$($repo):windows-nanoserver-latest",
+            "$($repo):windows-nanoserver-latest",
```

Comment:
Should we consider `windows-nanoserver-latest-lts` ?

Reply:
No harm in adding it I guess.
BTW this reminds we need to update the docker readme to reflect these changes.